### DISCUSSION TITLE
pending state condition

### DISCUSSION THREAD

**Discussion by haytxy** [grafana/grafana#39807] [Alerting]
hello, 

How do you configure alerts when an execution interval is hight and you have to wait for the next interval to check conditions for pending ?

here an example:  
How to disable pending state or how to work around it to be able not to wait for another 12h to execute check for pending condition

![image](https://user-images.githubusercontent.com/79607995/135307323-7bf19dd8-9a99-457b-89a6-5e82edbd2b47.png)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
